deploy/roles/gocd/defaults/main.yaml

```diff
@@ -1,10 +1,10 @@
 ---
 go_server_address: 10.0.1.26
 go_server_port: 8153
-gocd_agent_package_url: https://download.go.cd/binaries/16.10.0-4131/rpm/go-agent-16.10.0-4131.noarch.rpm
+gocd_agent_package_url: https://download.gocd.io/binaries/16.12.0-4352/rpm/go-agent-16.12.0-4352.noarch.rpm
 gocd_agent_package_name: go-agent-16.10.0-4131.noarch.rpm
```

Comment:
@mattboehm Interesting, they changed the URL once again :(  Anyways, the gocd_agent_package_name needs to match the URL, if you wouldn't ming updating that, thanks!